### PR TITLE
Improve security room settings tab style rules

### PR DESCRIPTION
**Pull request by luixxiul**
Fixes https://github.com/vector-im/element-web/issues/22575

|Before|After|
|---------|------|
|before|after|

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Improve security room settings tab style rules ([\#8844](https://github.com/matrix-org/matrix-react-sdk/pull/8844)). Fixes vector-im/element-web#22575. Contributed by @luixxiul.

### CODE REVIEW

File: res/css/views/settings/tabs/room/_SecurityRoomSettingsTab.scss [hunk 1]
```diff
@@ -16,24 +16,12 @@ limitations under the License.
...
-        margin-left: 3px;
-        margin-bottom: 5px;
+    .mx_SecurityRoomSettingsTab_warning {
+        display: flex;
+        align-items: center;
+        column-gap: $spacing-4;
     }
 }
-
-.mx_SecurityRoomSettingsTab_encryptionSection {
```
Comment (luixxiul) [inline res/css/views/settings/tabs/room/_SecurityRoomSettingsTab.scss:35]: This was added with https://github.com/matrix-org/matrix-react-sdk/pull/2523/files#diff-32beddecb1bd760a65b15adab68b83321fff88243061955848e3bca9c4554048R32, and this selector is no longer used.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Hi @luixxiul thank you for your contribution.

Can you fix the style lint errors and then re-request review please?

**Comment by luixxiul** [comment]
@weeman1337 Done. It looks like I have removed the last line break while fixing the conflict.

**Comment by luixxiul** [comment]
Also the spacing around the link is going to be adjusted with another PR to realize the same layout as Space setting's visibility tab.

**Comment by weeman1337** [comment]
@luixxiul can you please update this branch to develop? I will then give it another review on the Netlify deployment.

**Review by weeman1337 — APPROVED** [review]
Tested on the Netlify deployment. Thanks @luixxiul

**Pull request merged**